benchmarks/utils/monitor.py

```diff
@@ -165,7 +175,11 @@ def exec_func(*args, **kwargs) -> Dict[str, List[float]]:
                     if time:
                         t0 = perf_counter()
 
-                    func(*args, **kwargs)
+                    res_dic = func(*args, **kwargs)
+
+                    if not res_dic is None:
```

Comment:
Python style recommends
```suggestion
                    if res_dic is not None:
```